common/include/pcl/types.h

```diff
@@ -49,6 +49,8 @@
 
 #include <cstdint>
 
+#include <Eigen/StdVector>
```

Comment:
aligned_allocator is in `Eigen/Core`

Reply:
I'm sorry that I just mistaked the include library for `Eigen/StdVector` from [this page](https://eigen.tuxfamily.org/dox-devel/group__TopicStlContainers.html).
> In practice you must use the Eigen::aligned_allocator (not another aligned allocator), and #include <Eigen/StdVector>.

I'll fixed it right now. Thank you very much!